Defs/Stats/Stats_Weapons_Ranged.xml

```diff
@@ -78,6 +78,16 @@
 
   <!-- ========== Don't do anything, only display information to the player ========== -->
 
+  <StatDef>
+    <defName>MuzzleFlash</defName>
+    <workerClass>CombatExtended.StatWorker_MuzzleFlash</workerClass>
+    <label>muzzle flash</label>
+    <description>Blah blah blah.</description>
```

Comment:
'Programmer art' left in?  I believe this will show up on the weapon stat sheet, so enough detail for players to assess its impact is a good idea.

Reply:
No since i don't trust my english. I leave these parts to others.

Reply:
Fair.  I can provide a description, or a PR against your branch, or I can open a separate PR to fix the description after this one lands.

Reply:
just send it in discord

Reply:
`<description>How bright the muzzle flash is.\n\nDim muzzle flashes are enough to let enemy pawns target the shooter, while bright muzzle flashes illuminate the area around the shooter briefly.\nPawns will remember the position of the muzzle flash for 5-10 seconds.</description>`